### ISSUE TITLE
Custom dataset trained Mobile Detection Model v3  0 accuracy

### ISSUE THREAD

**Issue by saylan21** [PaddlePaddle/PaddleOCR#7553]
Hello everyone, 
I am trying to train **ch_PP-OCRv3_det_slim** model that shared on the Model List Page 
as the config file i am using [https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_cml.yml](url) the default .yaml file only changing the `**pretrained_model** as the weights i downloaded from model list v3 page
and using a custom dataset which has no problem while training recognation model.
Currently the model i downloaded from model list page, is working with 70 percent accuracy on my dataset, i would to increase the accuracy by training on this model weights with my own dataset but the training is ending up with 0 accuracy.
Could you tell me how to train this ch_PP-OCRv3_det_slim model with my own dataset and using the pretrained model on the model list page.
Thank you :)

**Comment by andyjpaddle** [comment]
if you want to finetune your own model, you should use this model `ch_PP-OCRv3_det` not slim model. also you should use detection datasets not recognition dataset. https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/dataset/ocr_datasets_en.md#1-text-detection

**Comment by saylan21** [comment]
Hi thank you for the answer, I am already using the detection dataset  which I created using PPOCRLabel tool. The model will run on a mobile device so I think it would be better to fine tune slim model. Is it possible to fine-tune slim model ?

**Comment by andyjpaddle** [comment]
It needs 2 step
first you finetune the base model (not slim)
second you  slim the model according this doc https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/deploy/slim/quantization/README_en.md#3-quant-aware-training

**Comment by andyjpaddle** [comment]
1. it depends on what your application scenarios are, we provide 3 fine-tune methods, but the  english doc is not ready, we'll update later, you can refer the Chinese doc https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_ch/PPOCRv3_det_train.md#3-%E5%9F%BA%E4%BA%8Eppocrv3%E6%A3%80%E6%B5%8Bfinetune%E8%AE%AD%E7%BB%83   Only change the dataset folder and the pretrained weight  is enough for cml method. The teacher model usually has a big model size, student model is little. you can choose that you need.
2. the input image size should be same with your training shape

**Comment by saylan21** [comment]
@andyjpaddle thank you for the answer 👍  i find it out that my custom trained inference model  is working better with a size multiple of 32. I also see this in the line 264 `https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/ppocr/data/imaug/operators.py` , Could you please tell me
1-) why it is required to resize my image to a multiple of 32 ? 
2-) why  It works better the more i upscale my image size ? For example it works better when i upscale my image size to 320 instead of 240 both of them are multiple of 32 ?

Thank you
